Tests/FluentAssertions.Specs/Primitives/DateTimeAssertionSpecs.cs

```diff
@@ -2245,6 +2245,24 @@ public void Should_support_chaining_constraints_with_and()
             // Assert
             action.Should().NotThrow();
         }
+
+        [Fact]
+        public void Should_throw_because_of_assertion_failure()
```

Comment:
I was about to suggest that we just extended an existing test that exercises a null subject with testing the failure message, but I see we don't have any such test 😞 
So it would be nice to also test this scenario for `After`